addons/binding/org.openhab.binding.knx/README.md

```diff
@@ -0,0 +1,283 @@
+| autoReconnectPeriod | N        | Seconds between connect retries when KNX link has been lost, 0 means never retry                             | 0             |
+
+## Things
+
+### *device* Things
+
+*basic* Things are wrappers around an arbitrary group addresses on the KNX bus. They have no specific function in the KNX binding, except that if the *address* is defined the binding will actively poll the Individual Address on the KNX bus to detect that the KNX actuator is reachable. Under normal real world circumstances, either all devices on a bus are reachable, or the entire bus is down. When *fetch* is set to true, the binding will read-out the memory of the KNX actuator in order to detect configuration data and so forth. This is however an experimental feature very prone to the actual on the KNX bus. 
+
+| Name         | Required | Description                                                                                                              | Default value                                                               |
+|--------------|----------|--------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------|
+| address      | N        | The individual device address (in 0.0.0 notation)                                                                        | -  |
+| fetch        | N        | Read out the device parameters and address/communication object tables (requires the address)                            | false                                                                       |
+| pingInterval | N        | Interval (in seconds) to contact the device and set the thing status based on the result (requires the address)          | 600                                                                         |
+| readInterval | N        | Interval (in seconds) to actively request reading of values from the bus (0 if they should only be read once at startup) | 0                                                                           |
+
+Different kinds of channels are defined and can be used to group together Group Addresses. All channel types share two configuration parameters: *read*, an optional parameter to indicate if the 'readable' group addresses of that Channel should be read at startup (default: false), and *interval*, an optional parameter that defines an interval between attempts to read the status group address on the bus, in seconds. When defined and set to 0, the interval is ignored (default: 0)
+
+#### Standard Channels
+
+Standard channels are used most of the time. They are used in the common case where the physical state is owned by a decive within the KNX bus, e.g. by a switch actuator who "knows" whether the light is turned on or of or by a temperature sensor which reports the room temperature regularly.
+
+##### Switch
```

Comment:
What does "Switch" refer to? To the ID of the channel type? If so, it should be "switch" (lowercase).
It is otherwise a bit confusing at the moment as the DSLs require all lower case.